video and images storage implemented

devJaja opened the pull request.

devJaja committed:
video and images storage implemented

Goodness5 reviewed (changes requested):
hi @devJaja please add a screenshot of the result of your implementation also please drop the route to your page

lastly please resolve the conflicts on your PR thanks

devJaja commented:
@Goodness5 
![Screenshot from 2025-01-27 15-44-50](https://github.com/user-attachments/assets/a8f6b6dc-d013-4ff0-b9db-a66c09f5c239)

devJaja commented:
@Goodness5 
this is the page /vidandimage_storage/file-storage

devJaja committed:
chore: responsive feat/file-storage

devJaja committed:
Merge branch 'master' of https://github.com/devJaja/Custos-Dapp into feat/vidandimage_storage

devJaja commented:
> hi @devJaja please add a screenshot of the result of your implementation also please drop the route to your page
> 
> lastly please resolve the conflicts on your PR thanks

@Goodness5 everything is resolved you can check it now

devJaja committed:
Merge branch 'master' into feat/vidandimage_storage

devJaja committed:
modified:   package.json
	modified:   public/fallback-development.js

devJaja committed:
modified:   package-lock.json
	modified:   package.json
	modified:   yarn.lock

devJaja commented:
conflict fixed

devJaja committed:
modified:   package-lock.json
	modified:   package.json
	modified:   yarn.lock

devJaja committed:
modified:   src/app/vidandimage_storage/SideBar

devJaja commented:
GM @Goodness5 
the error persisted on this branch 'vercel deployment fail' but i was able clone the repo then to do the work on different branch, 
I tried push to raise PR but the Permission to Custos-Diretriz/Custos-Dapp.git denied to devJaja.

devJaja committed:
Merge branch 'master' into feat/vidandimage_storage

Goodness5 commented:
> GM @Goodness5 the error persisted on this branch 'vercel deployment fail' but i was able clone the repo then to do the work on different branch, I tried push to raise PR but the Permission to Custos-Diretriz/Custos-Dapp.git denied to devJaja.

you get that error because you're trying to push directly to this repo you have to push to your forked version of it and create a PR requesting to merge to master not create a branch on this repo and wanting to merge that branch to master

devJaja committed:
deleted:    src/app/vidandimage_storage/Main.jsx
	renamed:    src/app/vidandimage_storage/BackNavigation.jsx -> src/app/vidandimage_storage/components/BackNavigation.jsx
	renamed:    src/app/vidandimage_storage/Header.jsx -> src/app/vidandimage_storage/components/Header.jsx
	renamed:    src/app/vidandimage_storage/SideBar.jsx -> src/app/vidandimage_storage/components/SideBar.jsx
	renamed:    src/app/vidandimage_storage/UploadOrRecord.jsx -> src/app/vidandimage_storage/components/UploadOrRecord.jsx
	modified:   src/app/vidandimage_storage/file-storage/page.jsx
	deleted:    src/app/vidandimage_storage/videorecorder/page.jsx
	modified:   yarn.lock

devJaja committed:
conflict fixed @ yarn.lock

devJaja committed:
Merge branch 'master' into feat/vidandimage_storage

Goodness5 committed:
Merge branch 'master' into feat/vidandimage_storage

devJaja committed:
Merge branch 'master' into feat/vidandimage_storage
